broker-plugin/broker-cli/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>io.enmasse</groupId>
+    <artifactId>broker-plugin</artifactId>
+    <version>0.30-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>broker-cli</artifactId>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.enmasse</groupId>
+      <artifactId>amqp-utils</artifactId>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>1.7.5</version>
```

Comment:
We use <dependencyManagement> section in the top level POM to globally control version numbers across the whole project.  You should remove the <version/> tag so you inherit from that.